uv.lock

```diff
@@ -1493,7 +1494,6 @@ wheels = [
 
 [[package]]
 name = "qontract-reconcile"
-version = "0.10.2.dev69"
```

Comment:
In my setup running `make test` locally without version produces an error.

Reply:
Which uv version are you running?

Reply:
also is it an error on the lockfile?

Reply:
uv version - 0.6.2

Yes.

```
error: Failed to parse `uv.lock`
  Caused by: TOML parse error at line 1495, column 1
     |
1495 | [[package]]
     | ^^^^^^^^^^^
missing field `version`
Error: building at STEP "RUN UV_DYNAMIC_VERSIONING_BYPASS="0.0.0" uv sync --frozen --no-group dev --python /usr/bin/python3": while running runtime: exit status 2
make: *** [Makefile:51: test-app] Error 2
```

Reply:
I've tried with uv version 0.6.10 and got the same error. Maybe this is something in my local setup. I'll investigate. Meanwhile LGTM